platform/darwin/scripts/generate-style-code.js

```diff
@@ -281,6 +281,30 @@ global.propertyDoc = function (propertyName, property, layerType, kind) {
             doc += `\n\nThis attribute corresponds to the <a href="https://www.mapbox.com/mapbox-gl-style-spec/#${anchor}"><code>${property.original}</code></a> layout property in the Mapbox Style Specification.`;
         }
     }
+    doc += '\n\nThis property can be set to one of the following values:\n\n' +
```

Comment:
We’re saying what kind of values the property can be set to, not which values specifically. Also, avoid passive voice:

> You can set this property to an instance of: